samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -0,0 +1,389 @@
+    document.getElementById('toggle-mic').disabled = true;
+
+    // Reset and disable the camera and mic selectors
+    const cameraSelector = document.getElementById('camera-selector');
+    const micSelector = document.getElementById('mic-selector');
+    cameraSelector.selectedIndex = 0;
+    micSelector.selectedIndex = 0;
+    cameraSelector.disabled = true;
+    micSelector.disabled = true;
+
+    // Update the call state in the UI
+    document.getElementById('camera-state').textContent = 'Camera: Off';
+    document.getElementById('mic-state').textContent = 'Mic: Off';
+    document.getElementById(
+      'participant-count'
+    ).textContent = `Participants: 0`;
+    document.getElementById(
+      'active-speaker'
+    ).textContent = `Active Speaker: None`;
+  }
+
+  /**
+   * Handles errors emitted from the Daily call object.
+   * @param {Object} e - The error event object.
+   */
+  handleError(e) {
+    console.error('DAILY SENT AN ERROR!', e);
+    if (e.error?.details?.sourceError) {
+      console.log('Original Error', e.error?.details?.sourceError);
+    }
+  }
+
+  /**
+   * Decides whether to display video or play audio based on the track type.
+   * @param {Object} event - The track event object.
+   */
+  displayTrack(event) {
+    if (event.track.kind === 'video') {
+      this.displayVideo(event);
+    } else {
+      this.playAudio(event);
+    }
+  }
+
+  /**
+   * Adds a <video> element to the DOM and attaches the video track to it.
+   * @param {Object} event - The track event object containing the video track.
+   */
+  displayVideo(event) {
+    const videosDiv = document.getElementById('videos');
+    const videoEl = document.createElement('video');
+    videosDiv.appendChild(videoEl);
+    videoEl.style.width = '100%';
+    videoEl.srcObject = new MediaStream([event.track]);
+    videoEl.play();
+  }
+
+  /**
+   * Adds an <audio> element to the DOM and attaches the audio track to it.
+   * @param {Object} event - The track event object containing the audio track.
+   */
+  playAudio(event) {
+    if (event.participant.local) {
+      return; // Avoid playing local audio
+    }
+    const audioEl = document.createElement('audio'); // Create <audio> element
+    document.body.appendChild(audioEl); // Add <audio> element to DOM
+    audioEl.srcObject = new MediaStream([event.track]); // Attach audio track to <audio> element
+    audioEl.play(); // Play audio track
+  }
+
+  /**
+   * Removes the track's element (audio or video) from the DOM when it stops
+   * and ensures media resources are released by setting srcObject to null.
+   * @param {Object} event - The track event object.
+   */
+  destroyTrack(event) {
+    let tracks = document.querySelectorAll('video, audio');
+    tracks.forEach((el) => {
+      if (el.srcObject && el.srcObject.getTracks().includes(event.track)) {
+        el.srcObject = null; // Release media resources
+        el.remove(); // Remove the element from the DOM
```

Comment:
One thought after running this, but may be out of scope so I will defer to you! Currently if you are testing with more than one participant and turn the camera off it can get a little confusing to see the visual representation of the other participant _completely_ disappear. We DO have the participant counter which helps, but another thing that might help is keeping a div for the participant with a background color, or just their session ID, or something to indicate another presence "tile" that is just toggled off. Totally optional like I said, might just be a little more intuitive with how people are used to seeing camera-off visuals in video calls.